webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/AssetExtractionService.java

```diff
@@ -466,7 +558,8 @@ Modifications getModifications(MultiBranchState base, MultiBranchState current)
 
         // we look into entries that are used in both, and we try to update it
 
-        ImmutableMap<String, Map.Entry<String, BranchData>> baseBothUsedBranchDataByMd5 = base.getBranchStateTextUnits().stream()
+        ImmutableMap<String, Map.Entry<String, BranchData>> baseBothUsedBranchDataByMd5 = base.getBranchStateTextUnits()
```

Comment:
A lot of change seems to be indentation, can we keep it out this commit for ease of reviewing?

Reply:
Done. IntelliJ's re-format file sabotaged me there :) .